app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -1,94 +1,14 @@
-    - label: Coronavirus (COVID‑19)
-      href: /coronavirus
-    - label: Crime, justice and the law
-      href: /browse/justice
-    - label: Disabled people
-      href: /browse/disabilities
-    - label: Driving and transport
-      href: /browse/driving
-    - label: Education and learning
-      href: /browse/education
-    - label: Employing people
-      href: /browse/employing-people
-    - label: Environment and countryside
-      href: /browse/environment-countryside
-    - label: Housing and local services
-      href: /browse/housing-local-services
-    - label: Money and tax
-      href: /browse/tax
-    - label: Passports, travel and living abroad
-      href: /browse/abroad
-    - label: Visas and immigration
-      href: /browse/visas-immigration
-    - label: Working, jobs and pensions
-      href: /browse/working
-- label: Departments
-  href: /government/organisations
-- label: Government activity
-  href: /search/news-and-communications
-  description: Find out what the government is doing
-  menu_contents:
-    - label: News
-      href: /search/news-and-communications
-      description: News stories, speeches, letters and notices
-    - label: Guidance and regulation
-      href: /search/guidance-and-regulation
-      description: Detailed guidance, regulations and rules
-    - label: Research and statistics
-      href: /search/research-and-statistics
-      description: Reports, analysis and official statistics
-    - label: Policy papers and consultation
-      href: /search/policy-papers-and-consultations
-      description: Consultations and strategy
-    - label: Transparency
-      href: /search/transparency-and-freedom-of-information-releases
-      description: Government data, freedom of information releases and corporate reports
-  footer_links:
-    - label: How government works
-      href: /government/how-government-works
-    - label: Get involved
-      href: /government/get-involved
-")
-
-popular_links = YAML::load("
-- label: 'Coronavirus (COVID-19): rules'
-  href: /guidance/covid-19-coronavirus-restrictions-what-you-can-and-cannot-do
-- label: 'Brexit: check what you need to do'
-  href: /brexit
-- label: Sign in to your personal tax account
-  href: /personal-tax-account
-- label: Find a job
-  href: /find-a-job
-- label: Sign in to your Universal Credit account
-  href: /sign-in-universal-credit
-")
-
 logo_link = "https://www.gov.uk/"
-logo_link_title = "Go to the GOV.UK homepage"
-logo_text = "GOV.UK"
-navigation_menu_heading = "Navigation menu"
-search_text = "Search GOV.UK"
+logo_link_title = t("components.layout_super_navigation_header.logo_link_title")
+logo_text = t("components.layout_super_navigation_header.logo_text")
+navigation_links = t("components.layout_super_navigation_header.navigation_links")
+navigation_menu_heading = t("components.layout_super_navigation_header.navigation_menu_heading")
+popular_links = t("components.layout_super_navigation_header.popular_links")
```

Comment:
Ah, now I see we moved these into the main locales. Do we want to do the same for the footer (in a separate PR)?

Reply:
Sounds good - though I'm unsure about whether this is the best place for this information. I think it's the best place for now - but could add it to the next frontenders meeting to see if people think that this is a good idea.

Reply:
Redesigning the footer is upcoming Explore Team work anyway. We can consider porting the microcopy as well then.